apps/builder/app/routes/rest.ai.op.ts

```diff
@@ -0,0 +1,145 @@
+import type { ActionArgs } from "@remix-run/node";
+import {
+  operations,
+  templateGenerator,
+  createGptModel,
+  type GPTModelMessageFormat,
+} from "@webstudio-is/ai";
+import { isFeatureEnabled } from "@webstudio-is/feature-flags";
+
+import env from "~/env/env.server";
+import { createContext } from "~/shared/context.server";
+
+const RequestSchema = operations.ContextSchema;
+
+export const maxDuration = 120;
+
+export const action = async function action({ request }: ActionArgs) {
```

Comment:
no need to use named functions, const is enough to have a fn.name in stack traces